### ISSUE TITLE
Show emoji/full names

### ISSUE THREAD

**Issue by nriley** [ZeeZide/Shrugs#11]
It seems Shrugs displays display names instead of full names and the names of the emoji rather than their images. I would prefer full names and the emoji in names be decoded. (This is equivalent to setting Preferences > Messages & media > Names > Full & display names in the Slack Web app.)

Here's an example:

Shrugs (don't want):
<img width="250" alt="image" src="https://user-images.githubusercontent.com/68303/84605898-1b3ff280-ae6f-11ea-88f5-b566c798e775.png">

Slack (do want):
<img width="144" alt="image" src="https://user-images.githubusercontent.com/68303/84605905-2abf3b80-ae6f-11ea-81a9-45305a6244e5.png">

**Comment by helje5** [comment]
Makes sense, including having a default to pick the preferred display. I can do the Emoji resolution (same like in the topic header, which already supports it). Both should be pretty easy to implement.
